src/main/java/com/salesforce/mirus/config/TaskConfigDefinition.java

```diff
@@ -38,13 +44,26 @@ static ConfigDef configDef() {
         ConfigDef.Type.STRING,
         "",
         ConfigDef.Importance.HIGH,
-        "The list of partitions for this task to handle");
+        "The list of partitions for this task to handle.");
     configDef.define(
         CONSUMER_CLIENT_ID,
         ConfigDef.Type.STRING,
         "",
         ConfigDef.Importance.HIGH,
-        "Client ID used to uniquely identify the consumer in this task");
+        "Client ID used to uniquely identify the consumer in this task.");
+    configDef.define(
+        REPLAY_POLICY,
+        ConfigDef.Type.STRING,
+        ReplayPolicy.IGNORE.toString(),
+        ConfigDef.Importance.LOW,
+        "Policy for handling bursts of duplicate records caused by offset resets. Allowed values: IGNORE, FILTER.");
```

Comment:
Need clearification: what IGNORE and FILTER stands for?
Records are checked with isSkippedRecord() if FILTER ReplayPolicy is set.
IGNORE is set by default and MirusSourceTask ignore all records all the time. Does it makes any sense?